corehq/apps/accounting/forms.py

```diff
@@ -223,6 +228,18 @@ def __init__(self, account, *args, **kwargs):
                 'pre_or_post_pay',
                 'account_basic',
                 crispy.Div(*additional_fields),
+                crispy.Field('billing_admin_emails', css_class='input-xxlarge accounting-email-select2'),
+                crispy.Div(
```

Comment:
Could you explain what this does?  I was thinking of making ```billing_admin_emails``` visible only when the ```is_customer_billing_account``` checkbox is selected, using knockout (like ```no_invoice_reason``` in ```SubscriptionForm```)

Reply:
Ah, that's my bad. `billing_admin_emails` isn't actually interactive because there's no `data_bind`. I'll take the `no_invoice_reason` example and update this div.